help/faq/texlive.md

```diff
@@ -0,0 +1,154 @@
+    
+    Users who experienced issues with these packages may wish to replace their documents in order to take advantage of the latest enhancements and bug fixes for these packages.
+    
+    fix: replace document to reprocess with latest versions of these packages.
+    
+
+MORE COMING SOON
+
+_Update 2011-12-06: arXiv switched to [texlive 2011](http://tug.org/texlive/) with a new, updated and enhanced tree of local addons. Most of the advice below still applies._
+
+As of 31 December 2009 arXiv is running [texlive 2009](http://tug.org/texlive/) with a new, updated and enhanced tree of local addons, for processing all new submissions and replacements. This represents a significant improvement over the aging teTeX system but will likely be slightly disruptive initially. Submitters may have to change some of their (bad) habits, and possibly adjust some TeX sources.
+
+The good news is that the arXiv installation has been kept very close to the official texlive-2009 system with very few local customizations and modifications. Texlive is one of the current comprehensive mainstream tex systems available for most platforms (and as live DVD). Therefore authors with up to date TeX system should encounter fewer surprises and problems
+
+An incomplete list of common symptoms, errors, and fixes follows:
+
+1.  [natbib](http://www.ctan.org/tex-archive/macros/latex/contrib/natbib/) was updated to version 8.31 \[2009/07/16 8.31 (PWD, AO)\]. There are some incompatibilities with previous version of natbib (arXiv's previous version was 7.1 \[2003/06/06 7.1 (PWD)\]. We recommend to adjust sources to use the current version of natbib. If all else fails, bundle your version of natbib.sty with your submission.
+    
+    The typical symptoms are
+    
+    ! Package natbib Error: Bibliography not compatible with author-year
+    citations.
+    
+    (natbib)                Press <return> to continue in numerical citation
+        style.
+    
+    See the natbib package documentation for explanation.
+    Type  H <return>  for immediate help.
+     ...
+    
+    l.125 ...and\\NAT@force@numbers{}\\NAT@force@numbers
+    
+    ?
+    ! Emergency stop.
+    
+    fix: The submission must be made compatible with natbib version 8.31
+    
+2.  aa.cls (6.1) and natbib.sty (8.3x collide). The symptoms are:
+    
+    ! LaTeX Error: Command \\bibfont already defined.
+                   Or name \\end... illegal, see p.192 of the manual.
+    
+    See the LaTeX manual or LaTeX Companion for explanation.
+    Type  H <return>  for immediate help.
+     ...
+    
+    l.15 \\begin{document}
+    
+    fix: do not include your own copy of aa.cls (v6.1). arXiv has a patched version which avoids the above problem.
+    
+3.  Bad size declared in binary blocks of PostScript files ( %%BeginBinary: .... count ). Symptoms:
+    
+    dvips: ! premature end of file in binary section
```

Comment:
same issue as tetex3, the dvips block gets codehilite into all dark green and bold. 
This page has some inconsistencies in whether subheadings are italic, bold, or whatnot, but that may be intentional emphasis.

Reply:
Will fix the code formatting later. Also need to reconcile duplicate pages in /help and /help/faq